### PR TITLE
Added docs in README.me. Fixes #55.

### LINKED ISSUE #55: It would be nice to have some documentation, README.md, docs
It could be my limited intelligence.

But I actually think there would be a lot of people who would prefer to have a little documentation, like some samples in the `README.md` to get them going more quickly using this library.

Now you have to get it all from the code itself, which just takes more time.

### PR DESCRIPTION
**Pull request by bartvanderwal**
A proposal. Please review.

### CODE REVIEW

**Comment by artyil** [comment]
+1

**Comment by axelson** [comment]
+1

**Comment by mkyral** [comment]
:+1:

**Comment by onzag** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -1,3 +1,86 @@
...
+
+    npm install exif-js --save    
+
+Or [Bower](http://bower.io/):
+
+    bower install exif-js --save
+
+Then add a `script` tag in your an HTML in the [best position](http://stackoverflow.com/questions/436411/where-is-the-best-place-to-put-script-tags-in-html-markup) referencing your local file.
+
+    <script src="vendors/exif-js/exif-js"></script>
```
Comment (anthu) [inline README.md]: there is a type , it propably should be:
<script src="vendors/exif-js/exif.js"></script>
(The last hyphen should be a dot)

**Comment by grekko** [comment]
Is there any author left working on this project?

**Comment by anthu** [comment]
As far as I can see the author deleted his account on GitHub.

**Comment by jseidelin** [comment]
I'm the original author. I tried to hand it over to someone else a while ago but I guess that failed.

I have no interest in this project any more and don't want to maintain it. If any of you want to take over, let me know and I'll give you access.

**Comment by bartvanderwal** [comment]
Great work @jseidelin.

If you give me access at least I'll accept my own pull request. ;)

This is a dependency in a project I'm working on. (Though it's currently on hold).

**Pull request merged**